nano/test_common/system.cpp

```diff
@@ -150,10 +150,8 @@ nano::test::system::system (uint16_t count_a, nano::transport::transport_type ty
 
 nano::test::system::~system ()
 {
-	for (auto & i : nodes)
-	{
-		i->stop ();
-	}
+	// Only stop system in destructor to avoid confusing and random bugs when debugging assertions that hit deadline expired condition
```

Comment:
This change should really be in a different PR.

Reply:
Probably should be, I just got annoyed at system stopping everything for no good reason and making it harder for me to debug things. It's a relatively small change though